src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * to represent commonalities from Phone, Email and Address
```

Comment:
The comment line should be in active tense. 
`Represents a person's contact information` may be more appropriate here.